src/state/index.ts

```diff
@@ -52,26 +54,59 @@ const persistConfig = {
   migrate: createMigrate(migrations, { debug: false }),
 }
 
-const persistedReducer = persistReducer(
-  persistConfig,
-  combineReducers({
-    farms: farmsReducer,
-    pools: poolsReducer,
-    predictions: predictionsReducer,
-    lottery: lotteryReducer,
-    info: infoReducer,
-    nftMarket: nftMarketReducer,
-
-    // Exchange
-    user,
-    transactions,
-    swap,
-    mint,
-    burn,
-    multicall,
-    lists,
-  }),
-)
+const allReducers = combineReducers({
+  farms: farmsReducer,
+  pools: poolsReducer,
+  predictions: predictionsReducer,
+  lottery: lotteryReducer,
+  info: infoReducer,
+  nftMarket: nftMarketReducer,
+
+  // Exchange
+  user,
+  transactions,
+  swap,
+  mint,
+  burn,
+  multicall,
+  lists,
+})
+
+const rootReducer = (state, action) => {
+  if (action.type === 'global/resetUserState') {
+    // eslint-disable-next-line no-param-reassign
+    state = {
```

Comment:
I think after cleaning the state, we would not run further reducers. Returning state here is good enough.

Reply:
Done

Reply:
It seems the Build Fix Commit removed the return keyword ? 😀

Reply:
Fixed :D